src/sentry/tagstore/snuba/backend.py

```diff
@@ -382,13 +382,17 @@ def get_group_event_filter(self, project_id, group_id, environment_id, tags):
 
         conditions = [[['tags[{}]'.format(k), '=', v] for (k, v) in tags.items()]]
 
-        result = snuba.raw_query(start, end, groupby=None, selected_columns=['event_id'], conditions=conditions, orderby='-timestamp', filter_keys=filters, limit=1000, referrer='tagstore.get_group_event_filter')
-        result = set(row['event_id'] for row in result['data'])
+        result = snuba.raw_query(
+            start, end, groupby=None, selected_columns=['event_id'],
+            conditions=conditions, orderby='-timestamp', filter_keys=filters,
+            limit=1000, referrer='tagstore.get_group_event_filter')
 
-        if not result:
+        event_id_set = set(row['event_id'] for row in result['data'])
```

Comment:
Seems like the `result` should be a object of some sort instead of a dictionary since the structure is consistent?

Reply:
Yeah should probably have a `SnubaResult` class with everything using proper property accessors. Add it to the TODO list I guess.